components/libwebsockets/port/lws_port.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-FileCopyrightText: 2020-2022 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "stdio.h"
+#include <libwebsockets.h>
+
+extern int __real_mbedtls_ssl_handshake_step(mbedtls_ssl_context *ssl);
+extern int __wrap_mbedtls_ssl_handshake_step(mbedtls_ssl_context *ssl);
```

Comment:
Not `extern`, right?
```suggestion
int __wrap_mbedtls_ssl_handshake_step(mbedtls_ssl_context *ssl);
```

Reply:
yes, not even necessary, removed :D